docs/.vuepress/components/Key.vue

```diff
@@ -61,7 +61,7 @@ export default {
             .split(/\s*\+(?!$)\s*/)
             .map(this.translateKey)
             .map(x => `<kbd>${x}</kbd>`)
-            .join(" + ")
+            .join(" ")
```

Comment:
Why not `&nbsp;` directly? Special unicode whitespace is hard to make out otherwise

Reply:
Actually, I must be going mad. I thought I put that.